app/ui-react/packages/ui/src/Customization/apiClientConnectors/create/ApiConnectorCreatorSelectMethod.tsx

```diff
@@ -128,9 +127,9 @@ export const ApiConnectorCreatorSelectMethod: React.FunctionComponent<IApiConnec
    * Callback for when one or more file uploads have been accepted.
    */
   const onUploadAccepted = (files: File[]): void => {
-    const fileExt = checkDocStyle(files[0].name);
+    const fileExt = formatFileName(files[0].name);
 
-    if (fileExt === 'wsdl' || fileExt.includes('?WSDL')) {
+    if (fileExt === 'wsdl' || fileExt.includes('?wsdl')) {
```

Comment:
Kind of the same code in 2 places, would it be better to create a function like `isWsdlFile(filename: string)`?